app/templates/all_questions/en/cyp_r1_all_questions_en.html

```diff
@@ -8,8 +8,8 @@
 {% block content %}
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <span class="govuk-caption-l">{% trans %}{{fund_title}}{% endtrans %}
-        {% trans %}{{round_title}}{% endtrans %}</span>
+        <span class="govuk-caption-l">{% trans %}{{fund_title}}{% endtrans %} {% trans %}{{round_title}}{% endtrans %}
```

Comment:
Does this line address the spacing issue for CYP fund and round title? @srh-sloan 

<img width="363" alt="image" src="https://github.com/communitiesuk/funding-service-design-frontend/assets/36962596/65adb69c-8f3a-4d47-8b23-700541ce4c11">

Reply:
Yes 👍